makecite/tests/test_cmdline.py

```diff
@@ -39,6 +40,23 @@ def test_bibtex():
         get_bibtex('some_fake_package')
 
 
+def test_bibtex_from_package():
+    # Try package that has a __bibtex__ attribute, but skip if package is not installed
+    try:
+        import matplotlib
```

Comment:
Can you replace this with "emcee" (since it's smaller to install), and add emcee to the dependencies with a comment "needed for testing __bibtex__"